server/country/api.js

```diff
@@ -47,6 +47,19 @@ module.exports.init = (app) => {
     }
   })
 
+  // Returns all regions from country_region table
+  app.get('/country/regions', async (req, res) => {
+    try {
+      const regions = await CountryService.getRegions()
+      const sortedRegions = regions
+        .sort((region1, region2) => (region1.name > region2.name ? 1 : -1))
+        .map((region) => region.regionCode)
```

Comment:
regions must be sorted client side, if user changes language they must be re-sorted